cli/tests/containerize.bats

```diff
@@ -37,7 +37,7 @@ env_setup() {
 # However, that directory is not writable
 # and thus fails to be deleted by bats as part of the test teardown.
 podman_cache_reset() {
-  echo "Resetting podman cache" >&3
+  # echo "Resetting podman cache" >&3
```

Comment:
I commented this out because it looks like it's useful for debugging these tests, but the output shouldn't be displayed in every test run.